src/mlpack/methods/random_forest/random_forest.hpp

```diff
@@ -167,6 +167,8 @@ class RandomForest
    * @param minimumLeafSize Minimum number of points in each tree's leaf nodes.
    * @param minimumGainSplit Minimum gain for splitting a decision tree node.
    * @param maximumDepth Maximum depth for the tree.
+   * @param warmStart When set to `true`, it adds `numTrees` new trees to the
+   *     existing random forest else a new forest is trained from scratch.
```

Comment:
I think technically you have to write "or else", @rcurtin correct me if I'm wrong, but I would probably go with "otherwise" instead anyway.

Reply:
Yeah, probably I'd choose `otherwise` here. :+1: